eng/CodeGeneration.targets

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
```

Comment:
This target file goes away when the last user of `TemporaryUsePreviousGeneratorVersion` is gone.